### PR TITLE
REL: 2.6.0

### PR DESCRIPTION
**Pull request by effigies**
@yasseraleman Please add yourself to https://github.com/nipreps/sdcflows/blob/master/.maint/CONTRIBUTORS.md if you would like to be listed as a contributor on our releases.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,3 +1,27 @@
+2.6.0 (November 09, 2023)
```
Comment (effigies) [inline CHANGES.rst]:
```suggestion
2.6.0 (November 10, 2023)
```

**Comment by effigies** [comment]
@oesteban @mgxd Any objections to a feature release? We've not needed a change in weeks.

**Comment by oesteban** [comment]
None from me :) Thanks for pushing it.

**Review by mgxd — APPROVED** [review]
lgtm

**Pull request merged**